react-ui/src/sprint/index.js

```diff
@@ -107,12 +110,13 @@ const SprintDetail = ({ sprintDetails, attributes, token, projectName }) => {
                     </div>
                 ) : null}
 
-                {/* <div>
+                {data && isMultiChart ? (
```

Comment:
Please add this condition as well.
**data["total_burndown"] === undefined**

Reply:
Resolved